DESCRIPTION

```diff
@@ -20,7 +20,7 @@ License: Apache License 2.0 | file LICENSE
 Encoding: UTF-8
 LazyData: true
 NeedsCompilation: no
-SystemRequirements: Java (>= 1.6)
```

Comment:
It should be >6, Java 1.6 is not supported.

Reply:
Updated to `>6`